cluster/config.go

```diff
@@ -87,6 +87,17 @@ func (c *ContainerConfig) extractExprs(key string) []string {
 	return exprs
 }
 
+// SwarmID extracts the Swarm ID from the Config.
+// May return an empty string if not set.
+func (c *ContainerConfig) SwarmID() string {
+	return c.Labels[namespace+".id"]
```

Comment:
Oh right, I found the `namespace` const here. Is it possible to be reused in `api/handlers` as well?